docs/notebooks/atmodel_tutorial.ipynb

```diff
@@ -0,0 +1,1693 @@
Ritter\",\"StevenJ.Nowlan\",\"C.J.C.H.Watkins\",\"L.Wu\",\"SebastianMika\",\"JosephSill\",\"RonaldCole\",\"Hwang-SooLee\",\"Ch.Tietz\",\"S.Gold\",\"D.Lippe\",\"AndresRodriguez\",\"ChristofSchofl\",\"BernhardScholkopf\",\"PedroA.d.F.R.Hojen-Sorensen\",\"YoshiyukiKabashima\",\"HarrisonMonFookLeong\",\"Jung-WookCho\",\"HenrikFredholm\",\"MoiseH.Goldstein\",\"LouisCeci\",\"LawrenceD.Jackel\",\"AnthonyZador\",\"CharlesFefferman\",\"AnnaCorderoy\",\"Alexandervon-zur-Muhlen\",\"RonaldJ.Williams\",\"GeorgSchnitger\",\"RonPapka\",\"BernardVictorri\",\"RobertFrye\",\"JosePrincipe\",\"E.Majani\",\"S.Finch\",\"ChristopherAssad\",\"T.Duong\",\"T.Ohmi\",\"Ming-TakLeung\",\"EeroP.Simoncelli\",\"R.C.Williamson\",\"AndreasG.Andreou\",\"EduardoSontag\",\"M.M.Hochberg\",\"DavidShahian\",\"GadiPinkas\",\"StefanKnerr\",\"IanParberry\",\"SebastianThmn\",\"VijayR.Konda\",\"JensKohlmorgen\",\"DavidWarland\",\"AndrewR.Webb\",\"GeoffreyHinton\",\"RolfEckmiller\",\"RuthErlanson\",\"HansHenrikThodberg\",\"FredRieke\",\"D.T.Lawrence\",\"JohnLazzaro\",\"StephenPickard\",\"ViktorGruev\",\"P.R.Montague\",\"Wan-PingChiang\",\"BarakA.Pearlmutter\",\"J.AnthonyMovshon\",\"ToshioInui\",\"Y.Bengio\",\"AnthonyJ.R.Heading\",\"AlexSmola\",\"EMoss\",\"DanielNissman\",\"JuergenFritsch\",\"SharadSinghal\",\"KatalinM.Gothard\",\"EveMarder\",\"DanaRon\",\"LinaL.E.Massone\",\"RitaVenturini\",\"GiacomoM.Bisio\",\"EtienneBarnard\",\"StephenA.Fisher\",\"PeterL.Bartlett\",\"DoinaPrecup\",\"EdwinLewis\",\"S.K.Riis\",\"TadHogg\",\"Y.Xie\",\"W.H.Zaagman\",\"VijaySamalam\",\"Antalvan-den-Bosch\",\"PatrickGallinari\",\"HayitK.Greenspan\",\"ShumeetBaluja\",\"HongC.Leung\",\"EdwinR.Hancock\",\"AliA.Minai\",\"YasuharuKioke\",\"PaulNachtigall\",\"ReinerLenz\",\"R.Erlanson\",\"AapoHyvarinen\",\"RamaChellappa\",\"StephanPareigis\",\"H.Drucker\",\"ReinholdMann\",\"JonathanA.Marshall\",\"KevinCummings\",\"TomasoPoggio\",\"S.Yasui\",\"AlanLapedes\",\"FrancesS.Chance\",\"SanjayBiswas\",\"StefanSchaal\",\"C.T.Abdallah\",\"JoshuaB.Tenenbaum\",\"J.Tani\",\"P.Anandan\",\"N.Barkai\",\"J.JeffreyMahoney\",\"LaurensLeerink\",\"AkayshaC.Tang\",\"TomJ.Richardson\",\"T.Delbruck\",\"MasazumiKatayama\",\"TobiDelbruck\",\"ChristianDarken\",\"HowardKaushansky\",\"TongZhang\",\"A.L.Yuille\",\"PatriceSimard\",\"TakeoYamashita\",\"KnutMoller\",\"JianWu\",\"Hui-H.Hsu\",\"ToddTroyer\",\"ShigemTanaka\",\"R.S.Peterson\",\"ArchismartRudra\",\"PatrickHaffner\",\"RalfHerbrich\",\"JamesA.Ritcey\",\"ThomasBrown\",\"P.Campbell\",\"DavidMcAllester\",\"PeterE.Latham\",\"ChristophBregler\",\"NirLevy\",\"ManuelSamuelides\",\"AlanH.Kramer\",\"Shih-ChengYen\",\"TonyJebara\",\"EricPostma\",\"RogerShepard\",\"AlexanderJ.Smola\",\"IdanSegev\",\"HalbertWhite\",\"WilliamFaller\",\"DanaZ.Anderson\",\"W.R.Softky\",\"Guo-ZhengSun\",\"StefanHeil\",\"NigelM.Allinson\",\"StephenJudd\",\"MarcusFrean\",\"FranklinR.Arethor\",\"KristinaJohnson\",\"AntonSchwartz\",\"WilliamW.Cohen\",\"AmyMcGovern\",\"EmadN.Eskandar\",\"GwendalLeMasson\",\"R.D.Puff\",\"WalterMe
```

Comment:
I meant to write "You can treat the tags on the posts as authors" not "as topics". Fixed.